talkvoter/v1_resources.py

```diff
@@ -1,22 +1,70 @@
 from flask_restful import Resource, Api
-from flask import Blueprint
+from flask import Blueprint, abort
+from sqlalchemy.sql.expression import func
 from marshmallow import ValidationError
-from webargs.flaskparser import use_args
-from .models import db
-from .serializers import VoteSchema
+from .models import db, Talk
+from .serializers import VoteSchema, TalkSchema
 from .constants import VoteValue
 
 
 api_bp = Blueprint('api_v1', __name__)
 api = Api(api_bp)
 
 
+class TalksResource(Resource):
```

Comment:
Very similar name to `TalkResources`, maybe use `TalksListResource`

Reply:
makes sense... changed